templates/Visua11ySettings.jsx

```diff
@@ -36,7 +30,7 @@ export default function Visua11ySettings(config) {
         <div className='visua11ysettings__group visua11ysettings__group-visualdisplay' role='group' aria-labelledby='visualdisplay'>
           {/* Should 'visua11ysettings__group-title' read as a title or is the lablled list enough? */}
           {config._groups.visualDisplay &&
-          <div className='visua11ysettings__group-title' id='visualdisplay' role="heading" aria-level="1">
+          <div className='visua11ysettings__group-title' id='visualdisplay' role="heading" aria-level="3">
```

Comment:
Does this jump from heading level one in the notify to heading level 3?

Reply:
Hello. Yes, that is correct. The main notify title has a heading role and a notify level of 1. The visua11y grouping titles have been changed to a level of 3.